### ISSUE TITLE
Feature: Ability to retain custom genres / styles

### ISSUE THREAD

**Issue by brendan-pike** [Marekkon5/onetagger#102]
99% of the time I'm happy to overwrite all existing tags with discogs.com.  But there are times where I do use some custom tags only meaningful to me.  In these situations it would be nice if there was a retain list.
Example:
Let's say in my genre retain list I have "wild west" among others.
Then a certain track has these existing tags in genre "rock; wild west".   And discogs has this track listed as only "classic rock".  With this feature I would expect the track genre to be updated to now have "classic rock; wild west" thus retaining my custom tags.

**Comment by Marekkon5** [comment]
For genres and styles there is an option in advanced settings - Merge Genres which should retain your original genres and styles. Reopen or comment if this doesnt work as intended or if you have more issues. Thanks

**Comment by brendan-pike** [comment]
Yes I realise that, however 99% of the time I actually prefer my genres to be changed (corrected).  There is only a few specific tags I would care to protect.

**Comment by Marekkon5** [comment]
Well in that case it would require a manual whitelist. Or now you can just clean your tags later in QT.

However this seems kinda very specific and/or tools like Mp3Tag would do that. So I'll not be adding it this release and discuss it later.

**Comment by brendan-pike** [comment]
Yes I understand its pretty niche. I didn't realise mp3tagger could do this, however I'm a linux user so its not supported, unless its wine friendly perhaps.

**Comment by Marekkon5** [comment]
Mp3Tag works fine in wine, also on Linux PuddleTag exists, although I've never used it.
